ember-scoped-css/package.json

```diff
@@ -14,23 +14,29 @@
     "addon-main.cjs"
   ],
   "exports": {
```

Comment:
changes here now reflect the `dist` directory, and everything references the built output, rather than some things being `src`, and some not